appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmRuntimeFileLogHandler.java

```diff
@@ -32,22 +32,20 @@
  * with the cloud logging agent.
  *
  */
-public class VmRuntimeFileLogHandler extends FileHandler {
-
+public class VmRuntimeFileLogHandler extends FileHandler implements SystemLogger {
```

Comment:
I wonder if these 2 handlers (this and `RequestLoggerHandler` - app/access) should not be located at `appengine-java-logging`. The reason is, I think they are also relevant to vanilla jetty and maybe should be configured in that level?